tests/model_registry/rest_api/utils.py

```
@@ -87,3 +89,16 @@ def register_model_rest_api(
         f"associated artifact: {model_artifact}"
     )
     return {"register_model": register_model, "model_version": model_version, "model_artifact": model_artifact}
+
+
+def validate_resource_attributes(
+    expected_params: dict[str, Any], actual_resource_data: dict[str, Any], resource_name: str
+) -> None:
+    errors: list[dict[str, list[Any]]]
+    if errors := [
+        {key: [f"Expected value: {expected_params[key]}, actual value: {actual_resource_data.get(key)}"]}
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)